src/test/java/org/opensearch/security/dlic/dlsfls/DlsTest.java

```diff
@@ -366,7 +366,7 @@ public void testDlsWithMinDocCountZeroAggregations() throws Exception {
 
         // Significant Text Aggregation is not impacted.
         // Non-admin user with setting "min_doc_count=0". Expected to only have access to buckets for dept_manager".
-        String query3 = "{\"aggregations\":{\"significant_termX\":{\"significant_terms\":{\"field\":\"termX.keyword\",\"min_doc_count\":0}}}}";
+        String query3 = "{\"size\":100,\"aggregations\":{\"significant_termX\":{\"significant_terms\":{\"field\":\"termX.keyword\",\"min_doc_count\":0}}}}";
```

Comment:
> This test inserts 17 documents and then does a query for matches on them checking the 'hits'. This hits have a default limit of 10... so this test was failing all the time because it depends if you got the right mix of 10/17 items in the output. By making this size much larger this should prevent this issue from happening again.
> 
> Always be careful with defaults!

_Originally posted by @peternied in https://github.com/opensearch-project/security/pull/2190#discussion_r1005744843_